stories/wmts.planet.stories.js

```diff
@@ -95,7 +94,6 @@ export const getMapWTmsLayersFactory = () => {
       width: 256,
```

Comment:
There's probably a typo in the name of this story - `getMapWTmsLayersFactory` -> should be `getMapWmtsLayersFactory`

Reply:
Fixed, thanks

Reply:
:+1: Resolving